.github/ci/before_cmake.sh

```diff
@@ -9,7 +9,11 @@ cd /tmp
 # check that we can compile USD from sources (only Focal)
 # see https://github.com/ignitionrobotics/sdformat/issues/869
 return_code=0
-if [ "$(lsb_release -r -s)" != "20.04" ]; then
+if [ "$(lsb_release -r -s)" == "20.04" ]; then
+  USD_VERSION="21.11"
+elif [ "$(lsb_release -r -s)" == "22.04" ]; then
+  USD_VERSION="22.11"
```

Comment:
Can https://github.com/gazebosim/gz-usd/issues/2 be closed now? The comment above can be updated too.